src/scss/constants.scss

```
@@ -1,11 +1,16 @@
-$primaryColor: var(--primary-color);
-$backgroundColor: var(--background-color);
+$primaryColor: var(--primary-color-deprecated);
+
+$accent--over: var(--primary-accent--over);
+$accent--under: var(--primary-accent--under);
+$content--over: var(--primary-content--over);
+$content--under: var(--primary-content--under);
```

Palette convention applies to SCSS constants **and** CSS variables i.e. the palette convention applies to palette, no matter where used. Possibly TS/DB in the future.

```suggestion
$accent--over: var(--accent--over);
$accent--under: var(--accent--under);
$content--over: var(--content--over);
$content--under: var(--primary--under);
```

---

fixed in 64de4d5d8b89aef7293806ca968effa1a35a5aef